File: 3BV2/PP/P1/Ejercicios/06-Polimorfismo/Java/Parte2/PAAE.java
public class PAAE implements Persona {
    private String id;
    private String nombre;
    private String CURP;
    private String domicilio;

    public PAAE(String id, String nombre, String CURP, String domicilio) {
        this.id = id;
        this.nombre = nombre;
        this.CURP = CURP;
        this.domicilio = domicilio;
    }

    @Override
    public String getID() {
        return this.id;
    }

    @Override
    public String getNombre() {
        return this.nombre;
    }

    @Override
    public String getCURP() {
        return this.CURP;
    }

    @Override
    public String getDomicilio() {
        return this.domicilio;
    }
}

File: 3BV2/PP/P1/Ejercicios/06-Polimorfismo/Python/Parte2/paae.py
from persona import Persona

class PAAE(Persona):
    def __init__(self, id, nombre, curp, domicilio):
        self.id = id
        self.nombre = nombre
        self.curp = curp
        self.domicilio = domicilio

    def get_id(self):
        return self.id

    def get_nombre(self):
        return self.nombre

    def get_curp(self):
        return self.curp

    def get_domicilio(self):
        return self.domicilio
